app/models/course/assessment/category.rb

```diff
@@ -44,6 +44,13 @@ def initialize_duplicate(duplicator, other)
     end
   end
 
+  # @return [Boolean] true if post-duplication processing is successful.
```

Comment:
Similarly here once we have more duplication code, maybe better to shift it into a concern for defining methods relating to duplication.

Reply:
Yeah, agree that we should do this, esp for the bigger models.